### PR TITLE
Configuration and documentation change

### PR DESCRIPTION
**Pull request by sudeeshjohn**
Updating documentation and configuration file.

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp-power-automation.github.io/pull/7#" title="Author self-approved">sudeeshjohn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp-power-automation.github.io).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp-power-automation.github.io/blob/main/OWNERS)~~ [sudeeshjohn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: mkdocs.yml
Comment (bpradipt) [inline mkdocs.yml:1]: @sudeeshjohn can we change the site_name to "Deploying OpenShift on IBM Power Systems"

File: README.md
Comment (bpradipt) [inline README.md:1]: typo: publish

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp-power-automation.github.io/pull/7#pullrequestreview-542791152" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp-power-automation.github.io/pull/7#" title="Author self-approved">sudeeshjohn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp-power-automation.github.io).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp-power-automation.github.io/blob/main/OWNERS)~~ [bpradipt,sudeeshjohn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
